dev/source/docs/building-setup-windows10.rst

```diff
@@ -29,8 +29,8 @@ WSL Setup steps
 
 .. tip::
 
-  You can run XWindows applications (including SITL) by installing `VcXsrv <https://sourceforge.net/projects/vcxsrv/>`_  on Windows. You will need to set the DISPLAY variable to point to your display: in WSL it is 127.0.0.1:0, in WSL2 it will be the ip address of the PC's network port and can be set by using the code below. Also in WSL2 you may have to disable the firewall for public networks, or create an exception in order for VcXsrv to communicate with WSL2:
-
+  You can run XWindows applications (graphical programs, including SITL) by installing `VcXsrv <https://sourceforge.net/projects/vcxsrv/>`_, `Cygwin X <https://x.cygwin.com/>`_ or `Xming <https://sourceforge.net/projects/xming/>`_ on Windows. You will need to set the DISPLAY variable to point to your display: in WSL it is 127.0.0.1:0, in WSL2 it will be the ip address of the PC's network port and can be set by using the code below. For more information regarding configuring your XLaunch Software , you can take reference from `here <https://itservices.cas.unt.edu/software/windows-x-server-vcxsrv>`_. Also in WSL2 you may have to disable the firewall for public networks, or create an exception in order for VcXsrv to communicate with WSL2:
```

Comment:
this creates a build error....cant have URL hotlinks with the same name within the same document...
```suggestion
  You can run XWindows applications (graphical programs, including SITL) by installing `VcXsrv <https://sourceforge.net/projects/vcxsrv/>`_, `Cygwin X <https://x.cygwin.com/>`_ or `Xming <https://sourceforge.net/projects/xming/>`_ on Windows. You will need to set the DISPLAY variable to point to your display: in WSL it is 127.0.0.1:0, in WSL2 it will be the ip address of the PC's network port and can be set by using the code below. For more information regarding configuring your XLaunch Software , you can refer to `this resource <https://itservices.cas.unt.edu/software/windows-x-server-vcxsrv>`_. Also in WSL2 you may have to disable the firewall for public networks, or create an exception in order for VcXsrv to communicate with WSL2:
```

Reply:
Got it .I will work on the changes. Thank You Sir!

Reply:
you can click commit suggestion to accept changes also

Reply:
Thank you, Sir! I have made the changes please take a look